### PR TITLE
Added NOTE and XXX tags.

### PR DESCRIPTION
**Pull request by designorant**
http://en.wikipedia.org/wiki/Comment_(computer_programming)#Tags

### CODE REVIEW

**Comment by amercier** [comment]
+1

Even better, I would have loved to have a new "infos" category with a blue bullet, because `INFO` and `XXX` may be not as important as warnings and errors.

**Comment by skj3gg** [comment]
@amercier I really think almost all of these fit into a info category, but the point of SL-annotations is to point them out as problems/errors (FIXME) or possible problems/warnings (NOTE, README).

@designorant I do think `XXX` should be switched to a warning. XXX (to me) means, "I do stuff but it's fragile/hard to follow". Thoughts on this?

A sidenote to think about, `TODO` means "I don't work, yet", which IMO means error.

**Comment by amercier** [comment]
@skj3gg +1 on SL-annotations's point being to point errors/warnings, not info. I don't quite agree with the meaning of `TODO` though. IMHO, it's more "This will need to be improved eventually, but I don't have time to do it yet. Let's ship as it is and work on this later.". While I think `FIXME` is clearly an error: "This MUST be fixed before shipping". That's my personal opinion though, and it's good SL-annotations is configurable.

**Comment by designorant** [comment]
I agree that the point of the linter is to point out warnings and errors. However, since the plugin already had INFO listed as a warning, I simply followed the pattern set by the author with INFO.

Personally, I have never come across INFO or README but I do occasionally see and use NOTE. I do not consider it a warning and I agree a new category could be a better fit (if goes along the purpose of linting at all). I would find blue dots along the gutter very helpful but [there is no such option with SublimeLinter for that](http://sublimelinter.readthedocs.org/en/latest/gutter_themes.html#standard-gutter-themes).

In terms of semantics, consider the following:

````
// FIXME This fails in some scenarios
// TODO Optimise this to handle more events
// XXX I have got no clue how this works but apparently it is needed
// NOTE See issue #1337 for reasons behind this
````

If SublimeLinter had another category for simple, informative comments, I would put NOTE in there although I could see a plase for TODO in there as well.

**Comment by grotewold** [comment]
Made the XXX change and added in v1.0.9 

Thanks!

**Comment by designorant** [comment]
Happy to help. Did you rewrite the history by any chance while merging this btw? I understand this is merged in a slightly unusual way, and I can safely delete my fork.

**Comment by grotewold** [comment]
I did a git apply-mail (am) it's how I typically apply patches when I'm merging patches that don't have all the changes I want. You should see your commit in the history and your fork is safe to delete.




Man page is https://www.kernel.org/pub/software/scm/git/docs/git-am.html

**Comment by designorant** [comment]
Thanks for clarifying!
